NOTICE

```diff
@@ -1,5 +1,5 @@
 Apache APISIX
-Copyright 2017-2021 The Apache Software Foundation
+Copyright 2017-2022 The Apache Software Foundation
```

Comment:
The beginning of Copyright is wrong? CC @tzssangglass

Reply:
yes, it's wrong, @hf400159 can you change it to `Copyright 2021-2022 The Apache Software Foundation`?